desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -254,6 +254,7 @@ def execute(self, notebook, snippet):
     result = connection.execute(statement)
 
     cache = {
+      'logs': [message for message in result.cursor.fetch_logs()] if result.cursor and hasattr(result.cursor, 'fetch_logs') else [],
```

Comment:
Move it one line up into a variable? (easier to understand)

Reply:
done